src/main/java/org/apache/commons/imaging/formats/webp/WebPImageParser.java

```diff
@@ -277,13 +279,14 @@ public void close() throws IOException {
 
         WebPChunk readChunk() throws ImagingException, IOException {
             while (sizeCount < fileSize) {
-                int type = read4Bytes("Chunk Type", is, "Not a Valid WebP File", ByteOrder.LITTLE_ENDIAN);
-                int payloadSize = read4Bytes("Chunk Size", is, "Not a Valid WebP File", ByteOrder.LITTLE_ENDIAN);
+                int type = read4Bytes("Chunk Type", is, "Not a valid WebP file", ByteOrder.LITTLE_ENDIAN);
+                int payloadSize = read4Bytes("Chunk Size", is, "Not a valid WebP file", ByteOrder.LITTLE_ENDIAN);
                 if (payloadSize < 0) {
                     throw new ImagingException("Chunk Payload is too long:" + payloadSize);
                 }
                 boolean padding = (payloadSize % 2) != 0;
-                int chunkSize = payloadSize + 8 + (padding ? 1 : 0);
+                int extraPadding = Math.addExact(8, (padding ? 1 : 0));
```

Comment:
The maximum value of `extraPadding` is 9, so there is no risk of overflow.

Furthermore, the name `extraPadding ` is somewhat misleading, since `padding` only refers to the padding at the end of the chunk.

Reply:
>The maximum value of extraPadding is 9, so there is no risk of overflow.

`9` as per specification? Or is there a constant somewhere?

Asking as we had security issues that had to be fixed (with some urgency, which is no fun) where users crafted images with custom metadata, many times invalid image formats.

So we just need to confirm if `9` is the maximum value, or if the user can somehow manipulate it.

>Furthermore, the name extraPadding is somewhat misleading, since padding only refers to the padding at the end of the chunk.

Sorry, I just changed that, pushing a new version now... it uses a function to avoid having to delcare variables.